frontend/appflowy_flutter/lib/ai/widgets/prompt_input/select_model_menu.dart

```diff
@@ -214,40 +215,45 @@ class _CurrentModelButton extends StatelessWidget {
         behavior: HitTestBehavior.opaque,
         child: SizedBox(
           height: DesktopAIPromptSizes.actionBarButtonSize,
-          child: FlowyHover(
-            style: const HoverStyle(
-              borderRadius: BorderRadius.all(Radius.circular(8)),
-            ),
-            child: Padding(
-              padding: const EdgeInsetsDirectional.all(4.0),
-              child: Row(
-                children: [
-                  Padding(
-                    // TODO: remove this after change icon to 20px
-                    padding: EdgeInsets.all(2),
-                    child: FlowySvg(
-                      FlowySvgs.ai_sparks_s,
-                      color: Theme.of(context).hintColor,
-                      size: Size.square(16),
-                    ),
-                  ),
-                  if (!model.isDefault)
+          child: AnimatedSize(
```

Comment:
Did the designer provide the design for it? If so, we can use the existing buttons.